src/app/space/create/deployments/services/deployments.service.ts

```diff
@@ -248,9 +248,9 @@ export class DeploymentsService implements OnDestroy {
     applicationId: string,
     desiredReplicas: number
   ): Observable<string> {
-    const url = `${this.apiUrl}${spaceId}/applications/${applicationId}/deployments/${environmentName}/control?podCount=${desiredReplicas}`;
+    const url = `${this.apiUrl}${spaceId}/applications/${applicationId}/deployments/${environmentName}?podCount=${desiredReplicas}`;
     return this.http.put(url, '')
```

Comment:
I'm not sure what changed but testing this locally I get
`{"errors":[{"code":"jwt_security_error","detail":"missing header \"Authorization\"","id":"1lQ61EOz","status":"401","title":"Unauthorized"}]}
`

I suppose we need to add the auth header to this request now?

Reply:
Hmm. It was just working for me as-is with a local UI and prod-preview WIT, and it works for me on the CD link above as well.

That looks to me more like the WIT backend is getting a 401 from whichever OSO API it's talking to rather than the UI getting a 401 on its request to WIT though...

Reply:
You'll need an auth header essentially everywhere; I'm surprised if it worked before - I don't see how it could.

Reply:
could it be a mixup between production and prod-preview in your environment?

Reply:
Hmm. I'm not sure how it's occurring, but I do actually see the `Authorization: Bearer ${token}` header on the requests sent by the app instance on that CD link. Perhaps there's a request interceptor outside of the Deployments area that is adding in the header for us. @dgutride or @joshuawilson ?

It's probably better practice to explicitly add the auth header in here so I have a patch ready for that, but I'll hold off on committing it for now until we know what's happening.

Reply:
@andrewazores all the other http requests in this file do set the header though so I feel like it's safe enough to go that route.